BentoKit/BentoKit/Components/MultilineTextInput.swift

```diff
@@ -29,40 +40,73 @@ extension Component.MultilineTextInput {
         let textView = UITextView().with {
             $0.isScrollEnabled = false
             $0.translatesAutoresizingMaskIntoConstraints = false
-            $0.setContentHuggingPriority(.cellRequired, for: .vertical)
             $0.setContentCompressionResistancePriority(.cellRequired, for: .vertical)
             $0.textContainerInset = .zero
             $0.textContainer.lineFragmentPadding = 0.0
         }
 
-        let placeholderLabel = UILabel()
-        var didFinishEditing: ((String) -> Void)?
+        fileprivate let placeholderLabel = UILabel().with {
+            $0.translatesAutoresizingMaskIntoConstraints = false
+        }
+        fileprivate let contentView = UIView()
+        fileprivate let sendButton = UIButton(type: .system).with {
```

Comment:
I don't know if `sendButton` is a correct name for component which should rather be generic.
I don't mind having the button but I think we can generalize naming here :)

Reply:
Well, it's `send` because tapping the button will invoke `didFinishEditing` and `resignFirstResponder`.

Reply:
Ok. However, I don't know if I would assume "send" will resginFirstResponder & call didFinishEditing only looking on the name. I think `button` would also be a good name or `finishEditingButton`. 

Up yo you, that's only a suggestion. I'm not 100% sure it's better ;)

Reply:
I'm going to remove the send button anyway and try to use component composition, so this will no longer be an issue.